client/components/domains/map-domain-step/index.jsx

```diff
@@ -80,12 +81,26 @@ class MapDomainStep extends React.Component {
 	}
 
 	render() {
-		const suggestion = get( this.props, 'products.domain_map', false )
-			? {
-					cost: this.props.products.domain_map.cost_display,
-					product_slug: this.props.products.domain_map.product_slug,
-			  }
-			: { cost: null, product_slug: '' };
+		const { currencyCode } = this.props;
+		const product = get( this.props, 'products.domain_map', false );
+		let cost, suggestion;
+
+		if ( product ) {
+			const productCost = product.cost;
+			cost = formatCurrency( productCost, currencyCode, { precision: 0 } );
+
+			if ( productCost - getCurrencyObject( productCost, currencyCode ).integer > 0 ) {
```

Comment:
Not sure whether it's worth doing this with `currencyCode`. :) You can easily check if a number is an integer or if it has decimals.

```
productCost % 1 != 0
```